packages/terra-date-picker/src/react-datepicker/PopupContainer.jsx

```diff
@@ -2,9 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const propTypes = {
+  /**
+   * Prop to determine whether or not the container height is bounded.
+   */
   isHeightBounded: PropTypes.bool,
+  /**
+   * Prop to determine whether or not the container width is bounded.
+   */
   isWidthBounded: PropTypes.bool,
+  /**
+   * A callback function to execute when a key is pressed.
+   */
   onKeyDown: PropTypes.func,
+  /**
+   * A component to be included in the popup container.
```

Comment:
Can be changed to "Components to be included in the popup container" since more than 1 component can be included.